AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/related.js

```diff
@@ -14,7 +14,7 @@ define([
         defaults: {
             template: 'Magento_AdobeStockImageAdminUi/grid/column/preview/related',
             filterChipsProvider: 'componentType = filters, ns = ${ $.ns }',
-            filterTitleSelector: '.admin__current-filters-title-wrap',
+            filterInputSelector: '#words',
```

Comment:
Hi, it's not a best practice depends on the element id. In general, it's not the best practice to use id in templates. Current frameworks reuse the template and sometimes we have the cases when template renders on the page a few times, and if we use ID we will get some untraceable issues because due to HTML rule, an ID can be presented only one time on the page.  Please take a is it possible to depend on html class instead of id